app/questionPage.html

```diff
@@ -46,16 +49,36 @@ <h3 id="timer"> 30m 00s </h3>
             </div>
             <div class="question-container col-md-8">
                 <div id="questions"></div>
+                <div id="question-nav" class="question-nav"></div>
                 <button type="button" id="finish" class="btn btn-success finish">Finish</button>
```

Comment:
duplicate id 'finish'

Reply:
Used the same id because we'll get rid of 'this one' later on.